src/wizard/wizard.scss

```diff
@@ -1,168 +1,137 @@
+  flex-direction: column;
+  @include iui-text(small);
+  @include themed {
+    color: t(iui-color-background-positive);
+  }
 
+  &.iui-current {
+    .iui-wizard-step-name {
       @include themed {
-        color: t(iui-color-background-positive);
-      }
-
-      &:hover {
-        text-decoration: none;
-      }
-
-      > .iui-wizards-step-title {
-        text-align: center;
-        @include themed {
-          color: t(iui-text-color-muted);
-        }
-      }
-
-      > .iui-wizards-step-circle {
-        align-items: center;
-        border-radius: 50%;
-        display: flex;
-        height: $iui-l;
-        width: $iui-l;
-        justify-content: center;
-        overflow-wrap: break-word;
-        padding: 0;
-        position: relative;
-        text-align: center;
-        z-index: 1;
-        user-select: none;
-        @include themed {
-          border: 1px solid t(iui-color-background-5);
-          background-color: t(iui-color-background-1);
-          color: t(iui-text-color-muted);
-        }
+        color: t(iui-color-foreground-positive);
       }
+      font-weight: $iui-font-weight-semibold;
     }
 
-    > .iui-wizards-step-track,
-    > .iui-wizards-step > .iui-wizards-step-track {
-      height: $iui-xxs;
-      position: absolute;
-      top: $iui-l * 0.5;
-      z-index: 0;
-
-      &.iui-wizards-step-track-before,
-      &.iui-wizards-step-track-after,
-      &.iui-wizards-step-track-main {
-        @include themed {
-          background-color: t(iui-color-background-5);
-        }
-      }
-
-      &.iui-wizards-step-track-before,
-      &.iui-wizards-step-track-after {
-        width: 50%;
-      }
-
-      &.iui-wizards-step-track-before {
-        left: 0;
-      }
-
-      &.iui-wizards-step-track-after {
-        right: 0;
-      }
-
-      &.iui-wizards-step-track-main {
-        display: block;
-        flex-basis: fill;
-        flex-grow: 1;
-        overflow: visible;
-        position: relative;
+    .iui-wizard-circle {
+      font-weight: $iui-font-weight-semibold;
+      @include themed {
+        color: t(iui-color-foreground-accessory);
+        border-color: t(iui-color-background-positive);
+        background-color: t(iui-color-background-positive);
       }
+    }
 
-      &.iui-wizards-step-track-filled {
-        @include themed {
-          background-color: t(iui-color-foreground-positive) !important;
-        }
+    .iui-wizard-track-content::before {
+      @include themed {
+        background-color: t(iui-color-foreground-positive);
       }
     }
+  }
 
-    > .iui-wizards-step-current {
-      > .iui-wizards-step-title {
-        @include themed {
-          color: t(iui-color-foreground-positive);
-        }
-        font-weight: $iui-font-weight-semibold;
-      }
+  &.iui-completed {
+    &.iui-clickable .iui-wizard-circle {
+      @include iui-focus(iui-focus-positive-box-shadow);
 
-      > .iui-wizards-step-circle {
+      &:hover {
+        cursor: pointer;
```

Comment:
weird place for this rule. It can be moved up because cursor automatically implies hover